MediaBrowser.Model/Dlna/ContentFeatureBuilder.cs

```diff
@@ -159,10 +159,10 @@ public static class ContentFeatureBuilder
             }
 
             // Time based seek is currently disabled when streaming. On LG CX3 adding DlnaFlags.TimeBasedSeek and orgPn causes the DLNA playback to fail (format not supported). Further investigations are needed before enabling the remaining code paths.
```

Comment:
If this is not the case anymore we should remove the comment.

Since I cannot test the changes someone else would need to chime in before we merge this.